SuperGoodSoft/solidus_taxjar
# Add button to backfill transactions

What is the goal of this PR?
---

Basic implementation of transaction backfill with naive logging

How do you manually test these changes? (if applicable)
---

1. Visit the TaxJar settings admin page
2. Click "Backfill transactions"
3. Verify all  _complete_ transactions have a corresponding record on your TaxJar account

Merge Checklist
---


Screenshots
---

![Screen Shot 2022-03-21 at 14 37 59PDT](https://user-images.githubusercontent.com/8933450/159368035-60399398-1835-4b2b-abee-2513c10cb6b4.png)
![Screen Shot 2022-03-21 at 14 37 05PDT](https://user-images.githubusercontent.com/8933450/159368038-eb535cba-6346-438c-a004-f39bfcd6a102.png)
